change/office-ui-fabric-react-2020-06-04-12-41-39-datepicker-add-wrapper-styles.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Should be minor since it's adding a new API.
```suggestion
  "type": "minor",
```